README.md

```diff
@@ -13,11 +12,8 @@
 **This is an example of how you may give instructions on setting up your project locally.**
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
```

Comment:
- [ ] Please kindly update this section of your `readme file` with explicit steps on how to clone and run your app locally, as this will be of help to non technical users. Steps like: 
`In your terminal, move the location you want to clone the project`,
`Enter git: clone git@github.com:kemigabocatherine/Awesome-books.git`,
`cd Awesome-books`,
`code .`,
`Right click in the index.html file opened VS code and click on open live server`...........